src/main/java/org/carlmontrobotics/commands/AutoMATICALLYGetNote.java

```diff
@@ -54,35 +47,62 @@ public void execute() {
     double forwardDistErrMeters = ll.getDistanceToNoteMeters(); 
     double strafeDistErrMeters = forwardDistErrMeters * Math.tan(angleErrRad);
     // dt.drive(0,0,0);
-    dt.drive(Math.max(forwardDistErrMeters * 2, MIN_MOVEMENT_METERSPSEC),
-        Math.max(strafeDistErrMeters * 2, MIN_MOVEMENT_METERSPSEC), Math.max(angleErrRad * 2, MIN_MOVEMENT_RADSPSEC));
-    // 180deg is about 6.2 rad/sec, min is .5rad/sec
 
-    if (LimelightHelpers.getTV(INTAKE_LL_NAME)) {
-      intake.motorSetIntake(0.5);
-    }
-    if(intake.intakeDetectsNote()) {
-      timer.start();
-    } else {
-      timer.stop();
-      timer.reset();
+    double forwardSpeed = 0;
+    double strafeSpeed = 0;
+    double rotationalSpeed = 0;
+
+    if (LimelightHelpers.getTX(INTAKE_LL_NAME) == 1) {
+      if (angleErrRad >= 0) {
+        forwardSpeed = Math.max(
+            forwardDistErrMeters
+                * SmartDashboard.getNumber("forward speed multiplier", 1.5),
+            MIN_MOVEMENT_METERSPSEC);
+        strafeSpeed = Math.max(
+            strafeDistErrMeters
+                * SmartDashboard.getNumber("strafe speed multiplier", 1.5),
+            MIN_MOVEMENT_METERSPSEC);
+        rotationalSpeed = Math.max(
+            angleErrRad
+                * SmartDashboard.getNumber("rotational speed multiplier", 2),
+            MIN_MOVEMENT_RADSPSEC);
```

Comment:
(assuming your above if statement is `tv == 1`)
if the robot still sees the note and its directly in line with the robot, the robot will still turn because of `Math.max()`. IMO, you shouldn't have to use `Math.max()` for turning if you are trying to make sure the robot goes "fast"

Reply:
wait i didn't see your later if statement, but imo you shouldn't have to use `Math.max()`